.github/workflows/alps2dot.yml

```
@@ -0,0 +1,187 @@
+        [ -s test-amazon.dot ] || exit 1
+        
+        # Test title labels
+        node dist/cli.js ../docs/bookstore/alps.xml -l title > test-title.dot
+        [ -s test-title.dot ] || exit 1
+        
+        # Test validation
+        node dist/cli.js validate ../docs/bookstore/alps.xml
+    
+    - name: Upload coverage to Codecov
+      if: matrix.node-version == 20
+      uses: codecov/codecov-action@v3
```

actionlint (1.7.7)

the runner of "codecov/codecov-action@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)